mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/view.jsp

```diff
@@ -53,74 +58,64 @@
 					<liferay-ui:message key="tasks" />
 				</h3>
 
-				<ul class="task-list">
+				<ul class="task-list" id="<portlet:namespace />todo-list">
 					<li>
+
 						<liferay-ui:message key="sample-task" />
 
 						<aui:button cssClass="delete-task" icon="icon-remove" size="large" />
 					</li>
 				</ul>
 
 				<div class="remaining-tasks-container">
-					<div class="remaining-tasks-count">
+					<div class="remaining-tasks-count" id="<portlet:namespace />remaining-tasks-count">
 						1
 					</div>
 
 					<div class="remaining-tasks">
+
 						<liferay-ui:message key="tasks-left" />
 					</div>
 				</div>
 
-				<div class="finished-task">
+				<div class="finished-task" id="<portlet:namespace />finished-task">
 					<ul class="finished-task-list"></ul>
 				</div>
 			</div>
 		</liferay-ui:section>
 
 		<liferay-ui:section>
 			<div class="task-history">
+
 				<h3>
 					<liferay-ui:message key="task-history" />
 
 					<i class="icon-book"></i>
 				</h3>
 
-				<div class="empty-task-history">
+				<div class="empty-task-history" id="<portlet:namespace />empty-task-history">
 					<liferay-ui:message key="no-tasks" />
 				</div>
 
-				<table class="history-table">
-					<thead>
-						<tr>
-							<th>
-								<liferay-ui:message key="id" />
-							</th>
-							<th>
-								<liferay-ui:message key="task-description" />
-							</th>
-							<th>
-								<liferay-ui:message key="time" />
-							</th>
-						</tr>
-					</thead>
-					<tbody>
-						<tr>
-							<td class="td-1"></td>
-							<td class="td-2"></td>
-							<td class="td-3"></td>
-						</tr>
-					</tbody>
-				</table>
+				<ul class="history-list" id="<portlet:namespace />history-list"></ul>
 			</div>
 		</liferay-ui:section>
 
 		<liferay-ui:section>
-			<div class="finished-tasks">
+			<div class="profile-page"></div>
+		</liferay-ui:section>
+
+		<liferay-ui:section>
+			<div class="contact-form">
+
 				<h3>
-					<liferay-ui:message key="finished-tasks" />
+
+					<liferay-ui:message key="contact-us" />
+
+					<i class="icon-envelope"></i>
 				</h3>
 
-				<ul class="finished-task-list"></ul>
+				<%@ include file="contact.jsp" %>
```

Comment:
You are including file that doesn't exist. There is only error in UI.

If you are including a JSP, the path should start with ```/``` and it should be a JSP**F** (JSP Fragment), so Intellij knows how to parse it.